easyconfigs/h/HTSlib/HTSlib-1.19.1-GCC-12.3.0.eb

```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
```

Comment:
Can you try using either HTSlib 1.18 or 1.20 instead of adding this version.

Reply:
using 1.18-GCC-12.3.0-fPIC with suffix to avoid clash with upstream